code/modules/jobs/job_types/other/merc_classes/hollowdragoon.dm

```diff
@@ -0,0 +1,61 @@
+/datum/advclass/mercenary/hollowdragoon
+	name = "Hollow Dragoon"
+	tutorial = "You rode out from Amber Hollow on your loyal steed, seeking coin from the wider reaches of Psydonia. With armour salvaged from fallen knights and a spear in hand, you will fight for anyone, for a price."
+	allowed_races = list(\
+		SPEC_ID_HOLLOWKIN,\
+		SPEC_ID_HARPY,\
+	) // Technically should be humens too, but hollow's deserve something special too
+	outfit = /datum/outfit/job/mercenary/dragoon
+	category_tags = list(CTAG_MERCENARY)
+
+	maximum_possible_slots = 5
+	cmode_music = 'sound/music/cmode/Combat_Dwarf.ogg'
```

Comment:
Just to check, intentional?

Reply:
I feel the music fit them well, so yes, intentional